README.md

```diff
@@ -20,7 +20,8 @@ The following dependencies need to be installed in order to run applications cre
 
 | Dependencies | Version        |
 | ------------ | -------------- |
-| NodeJS       | 18.16          |
+| NodeJS       | ^18.16         |
+| NPM          | 8.3.1 or later |
```

Comment:
is this the only additional requirement, or does it need to be installed in a specific way? In the migration guide we currently require Nods.js installed via NVM.